.travis.yml

```diff
@@ -6,7 +6,6 @@ python:
 
 env:
   global:
-    - DISPLAY=:99.0
```

Comment:
what did this do?

Reply:
This was needed to be able to run xfvc, open a browser window (though headless) and render stuff.